### PR TITLE
Android tiers fix

### PR DESCRIPTION
**Pull request by XeoNovaDan**
Also applies to any other race mods that add races with null recipes

### CODE REVIEW

**Comment by inexcode** [comment]
Thank you, looks great.
But I would prefer to only merge .cs file changes.
I hope it's okay for you if I close this PR, commit only your code changes, recompile it for myself and test it to make sure everything is okay?
After that I'll release your update on steam. What's the best way to credit you?

**Comment by XeoNovaDan** [comment]
Yep, that's absolutely fine. A lil' mention will suffice regarding crediting; not too fussy regarding anything extravagant since it is only a 2-liner!

**Comment by inexcode** [comment]
By the way, what were the side effects of this mod conflict? Probably would be useful to know for another mods.

**Comment by XeoNovaDan** [comment]
NullReferenceException caused by attempting to add the blood donation recipe to null recipes field, which'd result in a total mod config reset.

**Comment by inexcode** [comment]
UH-HUH!
Probably relevant to the [issue 6](https://github.com/inexcode/Woohooer/issues/6) at WooHooer repo, will dig to this later.
Thank you for the tip!

File: Source/Harmony/AttachRecepies.cs [hunk 1]
```diff
@@ -19,6 +19,8 @@ public static void Postfix()
             foreach (var humanoidRace in humanoidRaces)
             {
                 Log.Message("Adding in draw blood");
```
Comment (Mehni) [inline Source/Harmony/AttachRecepies.cs]: That's a Log.Message for every race added; about fifty lines of logspam on a heavily modded game.

```suggestion
                
```

**Comment by inexcode** [comment]
As this pull request changes binary and csproj files, will close and commit myself.

**Comment by inexcode** [comment]
Deployed on Steam.

**Pull request closed without merging**